cmd/gardenlet/app/app.go

```diff
@@ -389,6 +403,11 @@ func (g *garden) Start(ctx context.Context) error {
 		return err
 	}
 
+	log.Info("Creating new secret and managed resource required by DWD")
```

Comment:
Nit: Not everybody looking into the gardenlet logs might be immediately familiar with the abbreviation "DWD".

```suggestion
	log.Info("Creating new secret and managed resource required by dependency-watchdog")
```